src/backend/src/utils/utils.ts

```diff
@@ -24,3 +27,40 @@ export const convertStatus = (status: WBS_Element_Status): WbsElementStatus =>
 export const buildChangeDetail = (thingChanged: string, oldValue: string, newValue: string): string => {
   return `Changed ${thingChanged} from "${oldValue}" to "${newValue}"`;
 };
+
+const TOKEN_SECRET = process.env.TOKEN_SECRET || 'i<3security';
+
+// generate a jwt using the user's first and last name
+export const generateAccessToken = (user: { firstName: string; lastName: string }) => {
+  return jwt.sign(user, TOKEN_SECRET, { expiresIn: '12h' });
```

Comment:
It's generally best practice to do 1 hour tokens, but I can't see any reason to pick 1, 12, or 24 hours. Just tossing it out there for consideration.

Reply:
12 hours for now since we dont have auto refresh